img/p/pdffiller.com.svg

```diff
@@ -0,0 +1 @@
.08ZM37.456.139a.79.79 0 0 1 .78.641l.013.141v22.193a.786.786 0 0 1-.65.769l-.143.013h-2.688a.794.794 0 0 1-.71-.435l-.048-.118-.37-1.188c-1.416 1.307-3.223 2.08-5.844 2.08-5.512 0-8.042-3.269-8.042-9.42 0-5.586 2.62-9.3 8.042-9.3 1.93 0 3.626.563 4.944 1.5l.298.223.06-.06-.055-5.304a.782.782 0 0 1 .465-.72l.126-.043 3.62-.945a.803.803 0 0 1 .202-.026Zm68.603 5.568c5.183 0 8.52 2.91 8.52 8.378l-.01.377c-.012.28-.035.622-.07 1.026a.79.79 0 0 1-.79.714h-11.462c.268 2.615 2.055 3.762 5.54 3.762l.668-.003c.974-.013 1.358-.08 2.699-.397a3.59 3.59 0 0 0 .512-.174.799.799 0 0 1 .97.297l.07.13.945 2.297a.778.778 0 0 1-.437 1.019l-.071.025c-2.478.728-4.471 1.074-5.981 1.04-7.057-.166-10.217-3.5-10.217-9.437 0-5.38 2.949-9.053 9.115-9.053l-.001-.001Zm21.543.092c.348 0 .677.022.984.06l.348.056c.43.082.711.492.629.916l-.026.096-1.072 3.212a.795.795 0 0 1-.944.514l-.25-.056a6.678 6.678 0 0 0-1.159-.096c-2.8 0-3.783 1.853-3.783 4.91v7.713a.787.787 0 0 1-.793.782h-3.538a.787.787 0 0 1-.792-.782V7.052c0-.432.355-.782.793-.782h2.619a.79.79 0 0 1 .78.65l.395 2.29h.09c1.25-2.235 3.275-3.41 5.719-3.41v-.001Zm-54.37.984.135.023 3.687.974a.757.757 0 0 1 .555.601l.01.126v14.648c0 .37-.269.677-.625.74l-.137.012H73.17a.76.76 0 0 1-.75-.617l-.013-.135V7.533a.76.76 0 0 1 .691-.75h.134-.001ZM69.84 0a.782.782 0 0 1 .785.885l-.031.137-1.074 3.28a.79.79 0 0 1-.61.528l-.144.013H60.64V9.33h7.491c.383 0 .712.273.781.65l.011.14-.03 2.952a.786.786 0 0 1-.652.762l-.141.013h-7.46v9.267a.786.786 0 0 1-.65.77l-.143.013h-3.582a.79.79 0 0 1-.78-.642l-.013-.14V2.085c0-1.084.838-1.974 1.91-2.076l.203-.01h12.256ZM50.213.108c.528 0 1.075.036 1.6.097.253.03.561.078.925.146l.096.025a.752.752 0 0 1 .517.817l-.032.132-.53 1.593a.762.762 0 0 1-.784.515l-.098-.007a11.463 11.463 0 0 0-.67-.02c-2.089 0-2.9.67-2.975 1.73l-.007.172v.564h3.786a.79.79 0 0 1 .78.641l.013.141v2.062a.786.786 0 0 1-.65.77l-.143.011h-3.786v13.618a.786.786 0 0 1-.65.769l-.142.013h-3.597a.79.79 0 0 1-.78-.642l-.012-.14V9.497h-1.919a.79.79 0 0 1-.78-.64l-.013-.141V6.654c0-.383.28-.703.65-.769l.143-.013h1.888v-.564c0-2.764 2.109-5.2 7.169-5.2h.001Zm39.709 2.347c.07 0 .14.01.206.028l3.597.955c.345.089.586.399.587.754v18.92a.787.787 0 0 1-.793.781h-3.597a.788.788 0 0 1-.793-.782V3.237c0-.432.356-.782.793-.782Zm-8.339 0c.07 0 .14.01.206.028l3.597.955c.344.089.585.399.587.754v18.92a.787.787 0 0 1-.792.781h-3.598a.788.788 0 0 1-.793-.782V3.237c0-.432.355-.782.793-.782ZM29.302 9.884c-3.042 0-4.187 1.456-4.187 5.14 0 3.804.934 5.29 4.188 5.29 1.92 0 3.073-.731 3.63-1.435l.105-.14v-7.37c-.693-.861-2.02-1.485-3.736-1.485ZM8.946 9.397c-1.92 0-3.073.73-3.63 1.435l-.105.14v7.37c.693.86 2.018 1.485 3.735 1.485 3.043 0 4.188-1.456 4.188-5.14 0-3.804-.934-5.29-4.188-5.29Zm97.256.34c-2.651 0-3.724 1.087-3.962 3.38h7.774c-.06-2.293-1.49-3.38-3.812-3.38ZM73.228.626l.136.024 3.683.974c.287.075.504.31.556.6l.01.126V4.4a.761.761 0 0 1-.825.75l-.135-.023-3.684-.974a.757.757 0 0 1-.554-.601l-.012-.126V1.378a.76.76 0 0 1 .691-.75h.134Z"/></g></g></svg>
```

Comment:
Thanks for your pull request!
You attached their logo, but we're looking for their icon. An icon is an image that's made to look good in smaller sizes, such as the favicon on a browser. As the space allotted to the icons are just 32 pixels, it cannot contain text as that wouldn't be visible to the user.
Here's pdfFiller's icon: 
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 133 99.5"><path fill="#e8501f"d="m112.2 0-57 99.5h41.7l36.1-63ZM74 59.8 65.6 44H23.8l29.4 52ZM108.3 0H0l20.8 36.4h66.6"/></svg>
```